### ISSUE TITLE
Inline Fluid Provider occasionally requests more trains than specified

### ISSUE THREAD

**Issue by DBosley** [Opinionated-Blueprints/10-Books-Full-of-Rails#76]
### Have you checked the following resources?

- [X] [Wiki](https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki)
- [X] [Discussions Forum](https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/discussions)
- [X] [Open and Closed Issues](https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/issues)

### Version of the Book

2.19.0

### Bug Description

Inline Fluid Provider occasionally requests more trains than specified

### How to reproduce the problem

In attached save, the sulfuric acid fluid provider will occasionally request a second train while one is already sitting there. This can cause backups in the train network.

[Buggy Fluid Provider - Sulfuric Acid.zip](https://github.com/user-attachments/files/17878195/Buggy.Fluid.Provider.-.Sulfuric.Acid.zip)

### What did you expect?

I expect the train limit to be respected

### What happened instead?

The train limit is not always respected

**Comment by mkaito** [comment]
I've had the "occasionally calls too many trains" happen with solid providers too. Every couple hours I'll find a random deadlock with 20 trains waiting behind a limit-1 station that's full and not going anywhere.

**Comment by Bocian-1** [comment]
I couldn't tell how that happened, but I reworked the wiring on inline stations a bit and providers now have a constant train limit of 1, so this shouldn't happen any more.

On the other hand, the setup of the requester doesn't match
![image](https://github.com/user-attachments/assets/d09b0e70-7e50-4838-a4aa-c1fee3342fbc)
It was probably made after a change to a few values a few updates ago, where I forgot to relabel them correctly for the fluid stations. It's better to remake it and other fluid stations as well.

@mkaito 
Was it also with inline providers? If so, then it also should be fixed with the above change.

It could be that when trying to fit the combinators on the inline stations I placed some too far apart and a wire was cut. Can't tell now.
